src/python/turicreate/toolkits/object_detector/object_detector.py

```diff
@@ -565,6 +568,159 @@ def wait_for_batch():
             if k in net_params:
                 net_params[k].set_data(mps_net_params[k])
 
+    elif params['use_tensorflow']:
```

Comment:
Maybe we should move this before the MPS case, on the theory that if the user explicitly requested TF, we shouldn't let the automatically computed `use_mps` override that...